src/styles.scss

```diff
@@ -118,6 +118,11 @@ body {
   position: relative;
 }
 
+.staging-banner {
+  background-color: #FFEB3B;
```

Comment:
If not used in any other component, this can safely be moved into the banner component's scss.  Also, bootstrap's bg-warning color looks preferably in my opinion.

Reply:
Sadly position: relative is a bootstrap 4-only class